src/main/java/tools/jackson/core/JsonGenerator.java

```diff
@@ -50,6 +50,19 @@ protected JsonGenerator() { }
     @Override
     public abstract Version version();
 
+    /*
+    /**********************************************************************
+    /* Constraints violation checking
+    /**********************************************************************
+     */
+
+    /**
+     * Get the constraints to apply when performing streaming writes.
+     */
+    public StreamWriteConstraints streamWriteConstraints() {
+        return StreamWriteConstraints.defaults();
```

Comment:
We can probably leave this abstract, similar to how `JsonParser` leaves matching method, to avoid possibility of something not overriding it.
(can't do in 2.x for compatibility reasons etc)

I can do this post-merge, just noting it here for posterity.